src/index.html

```diff
@@ -8,6 +8,33 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Product cards</h1>
+  <div class="card" data-qa="card">
+    <img
+      src="images/imac.jpeg"
+      alt="photo of iMac"
+      class="card--photo"
+      >
```

Comment:
закрывающая стрелка должен быть на одной ширине с открывающей
```suggestion
    >
```